packages/circuit-ui/package.json

```diff
@@ -87,6 +88,7 @@
     "react-dom": "^17.0.1",
     "react-svg-loader": "^3.0.3",
     "react-swipeable": "^6.1.0",
+    "react-use": "^17.2.4",
```

Comment:
Oh no, I didn't see this 😭 

I don't think it makes sense to keep `react-use` then, especially not as a peer dependency, because when we migrate away, other apps that will have started to depend on it will have to migrate too.

I would either:
1. use `react-use` as a "normal" dependency just for the Popover/Modal for now, and migrate away in 3.x (it shouldn't be a breaking change for apps)
2. contribute a `useClickOutside` to `react-hookz`, but I'm not sure how long this would take and if it fits our v3 timeline.

I tend to favor solution 1 now, and reevaluate `react-hookz` in v3.x (also checking alternatives) and possibly contribute missing hooks there.

What do you think?

Reply:
I also prefer option 1.